sessions/big-shiny-apps/shinytest2-unit-testing-for-shiny.md

```diff
@@ -8,11 +8,11 @@ session_slug: big-shiny-apps
 sched_url: https://rstudioconf2022.sched.com/event/11ibY
 # ---- Edit information below this line ----
 # The title of your talk
-talk_title: "{shinytest2}: Unit testing for Shiny applications"
+talk_title: "{shinytest2}: Regression testing for Shiny applications"
```

Comment:
We just finalized signage yesterday evening, so to avoid confusion during the conf we're going to hold off on title adjustments. I'll move it to a new PR we could merge after conf.

```suggestion
talk_title: "{shinytest2}: Unit testing for Shiny applications"
```